prodigy/analysis/instruction_handler.py

```diff
@@ -483,7 +483,8 @@ def compute(
         rhs_block, rhs_error_prob = SequenceHandler.compute(
             instruction.rhs, prog_info, distribution, error_prob, config)
         logger.info("Combining PChoice branches.\n%s", instruction)
-        new_prob = lhs_error_prob * str(instruction.prob) + rhs_error_prob * f"1-({instruction.prob})"
+        # TODO this var was never used, can this be deleted?
```

Comment:
It depends. I think we currently just allow Query commands at the end of the program. If program instructions can follow afterwards, we need to pass through the correct distribution and error probabilities. 

Also what should happen when we query somewhere inside the program but the distribution is not yet normalised?

Reply:
This line was commented due to lint complaining that the assigned variable `new_prob` was never used. Because I was not sure what this variable *should* do if used, I wanted to mark this somehow before deleting something